sonar-plugin-api/src/main/java/org/sonar/api/test/TestPlan.java

```diff
@@ -22,8 +22,9 @@
 import org.sonar.api.component.Perspective;
 
 /**
- * A {@link TestPlan} is 
+ * @deprecated since 5.2 not used
  */
+@Deprecated
 public interface TestPlan<T extends TestCase> extends Perspective {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png)Do not forget to remove this deprecated code someday.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)